tests/suites/test_suite_gcm.aes128_en.data

```diff
@@ -670,6 +670,58 @@ AES-GCM NIST Validation (AES-128,128,1024,1024,32) #2 [#2]
 depends_on:MBEDTLS_AES_C
 gcm_encrypt_and_tag:MBEDTLS_CIPHER_ID_AES:"fe481476fce76efcfc78ed144b0756f1":"246e1f2babab8da98b17cc928bd49504d7d87ea2cc174f9ffb7dbafe5969ff824a0bcb52f35441d22f3edcd10fab0ec04c0bde5abd3624ca25cbb4541b5d62a3deb52c00b75d68aaf0504d51f95b8dcbebdd8433f4966c584ac7f8c19407ca927a79fa4ead2688c4a7baafb4c31ef83c05e8848ec2b4f657aab84c109c91c277":"1a2c18c6bf13b3b2785610c71ccd98ca":"b0ab3cb5256575774b8242b89badfbe0dfdfd04f5dd75a8e5f218b28d3f6bc085a013defa5f5b15dfb46132db58ed7a9ddb812d28ee2f962796ad988561a381c02d1cf37dca5fd33e081d61cc7b3ab0b477947524a4ca4cb48c36f48b302c440be6f5777518a60585a8a16cea510dbfc5580b0daac49a2b1242ff55e91a8eae8":"5587620bbb77f70afdf3cdb7ae390edd0473286d86d3f862ad70902d90ff1d315947c959f016257a8fe1f52cc22a54f21de8cb60b74808ac7b22ea7a15945371e18b77c9571aad631aa080c60c1e472019fa85625fc80ed32a51d05e397a8987c8fece197a566689d24d05361b6f3a75616c89db6123bf5902960b21a18bc03a":32:"bd4265a8":0
 
+AES-GCM NIST CAVS 14.0 - empty plaintext, AD length: 128 bytes, ciphertext updates: 0
```

Comment:
For future reference, our test framework scripts truncate test case descriptions at 66 characters, so it's better if they fit in 66 characters, and annoying when the first 66 characters aren't unique.
```
AES-GCM NIST CAVS 14.0 - empty plaintext, AD length: 128 bytes, ci  PASS
AES-GCM NIST CAVS 14.0 - empty plaintext, AD length: 128 bytes, ci  PASS
AES-GCM NIST CAVS 14.0 - empty plaintext, AD length: 128 bytes, ci  PASS
AES-GCM NIST CAVS 14.0 - empty plaintext, AD length: 90 bytes, cip  PASS
AES-GCM NIST CAVS 14.0 - empty plaintext, AD length: 90 bytes, cip  PASS
AES-GCM NIST CAVS 14.0 - empty plaintext, AD length: 90 bytes, cip  PASS
AES-GCM NIST CAVS 14.0 - empty AD, plaintext length: 128 bytes, AD  PASS
AES-GCM NIST CAVS 14.0 - empty AD, plaintext length: 128 bytes, AD  PASS
AES-GCM NIST CAVS 14.0 - empty AD, plaintext length: 128 bytes, AD  PASS
AES-GCM NIST CAVS 14.0 - empty AD, plaintext length: 51 bytes, AD   PASS
AES-GCM NIST CAVS 14.0 - empty AD, plaintext length: 51 bytes, AD   PASS
AES-GCM NIST CAVS 14.0 - empty AD, plaintext length: 51 bytes, AD   PASS
```
It's a preexisting problem in many test suites, and we're pressed for time, so I'm not going to block the PR on this.